graphql/mappers/profile.js

```diff
@@ -5,10 +5,37 @@ import { buildLink } from '../../lib/links'
 export async function getProfileContent() {
   const query = require('../queries/profile.graphql')
   const response = await clientQuery(query)
+
+  const enLookingForFragment = findFragmentByScId(
+    response,
+    'looking-for-security-settings'
+  ).scContentEn
+  const frLookingForFragment = findFragmentByScId(
+    response,
+    'looking-for-security-settings'
+  ).scContentFr
+
   const mappedProfile = {
     en: {
+      breadcrumb:
+        response.data.schPagev1ByPath.item.scBreadcrumbParentPages.map((w) => {
+          return {
+            link: w.scPageNameEn,
+            text: w.scTitleEn,
+          }
+        }),
       pageName: response.data.schPagev1ByPath.item.scTitleEn,
       heading: response.data.schPagev1ByPath.item.scTitleEn,
+      lookingFor: {
+        title: enLookingForFragment.json[0].content[0].value,
+        subText: enLookingForFragment.json[1].content.map((element) => {
+          if (element.value) {
+            return element.value
+          }
+          return null
```

Comment:
This logic could be simplified to `return element.value || null` I think :thinking: